Frameworks/Foundation/NSOperationQueue.mm

```diff
@@ -196,6 +196,12 @@ - (NSOperationQueue*)_doMainWork {
         didWork = FALSE;
 
         for (int i = 0; i < NSOperationQueuePriority_Count; i++) {
+            [priv->suspendedCondition lock];
+            while (priv->isSuspended) {
+                [priv->suspendedCondition wait];
+            }
```

Comment:
Hooray saying the same things at roughly the same time.

---
In reply to: [69806160](https://github.com/Microsoft/WinObjC/pull/613#discussion_r69806160) [](ancestors = 69806160,69805759)